modules/ROOT/pages/push/proc_preparing-ups-for-use-with-ios.adoc

```diff
@@ -0,0 +1,109 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: proc_preparing-ups-for-use-with-ios
+
+[id='{context}_preparing-ups-for-use-with-ios']
+= Preparing {unifiedpush-service} For Use With iOS
+
+Before using {unifiedpush-service} with iOS, you must complete the following actions:
+
+* Request a Signed Certificate
+* Create an Apple App ID and SSL Certificate for APNs
+
+[discrete]
+== Prerequisites
+
+* An Apple developer account
```

Comment:
Any other prerequisites?

Reply:
You need an Apple Computer for development and a Apple device for testing real notifications.

Reply:
I think that for this preparation, iOS device is not needed, so I have removed it from prerequisites in my last commit.